orttraining/orttraining/python/training/ortmodule/_utils.py

```diff
@@ -50,6 +51,33 @@ def _ortvalue_to_torch_tensor(ortvalue, device):
     dlpack_tensor = ortvalue.to_dlpack()
     return _torch_tensor_from_dl_pack(dlpack_tensor, ortvalue, device)
 
+
+def _ortvalues_to_torch_tensor(ortvalues, device):
+    if len(ortvalues) == 0:
+        return tuple()
+    if (hasattr(ortvalues, 'to_dlpack') and
+            all(ortvalue.data_type() != 'tensor(bool)' for ortvalue in ortvalues)):
+        if device == 'ort':
+            return tuple(ortvalues.to_dlpack(
+                lambda dlpack_structure: C_OrtValue.from_dlpack(dlpack_structure, False)))
```

Comment:
Should be C.ort_from_dlpack here according to the _torch_tensor_from_dl_pack. C.ort_from_dlpack returns torch tensor, while C_OrtValue.from_dlpack returns OrtValue.